### PR TITLE
add Api route

### PR DESCRIPTION
**Pull request by zsluedem**

### CODE REVIEW

File: rchain-coop/src/routes/api.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  return query.replace("$addr", address)
+}
+
+router.get('/total-supply', function (req, res, next) {
+  res.json(TOTAL_SUPPLY);
+});
+
+router.get('/total-circulation', apiCache.middleware(CIRCULATION_CACHE_TIMEOUT), function (req, res, next) {
+  request.post(MAINNET_OBSERVER + '/api/explore-deploy', { body: totalCirculationQuery }, (err, resp, body) => {
+    if (resp.statusCode >= 400) {
```
Comment (tgrospic) [inline rchain-coop/src/routes/api.js]: Is there a reason to accept anything except 200 OK response from RNode?

File: rchain-coop/src/routes/api.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      res.send(resp.statusCode)
+    } else {
+      try {
+        const result = JSON.parse(body)
+        const circulation = result['expr'][0]['ExprInt']['data'] / REV_TO_PHLO
+        res.json(circulation)
+      }
+      catch{
+        res.send(400)
```
Comment (tgrospic) [inline rchain-coop/src/routes/api.js]: Maybe http _500 Server Error_ would be better here.

File: rchain-coop/src/routes/api.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      }
+    }
+  })
+});
+
+router.get('/balance/:address', apiCache.middleware(BALANCE_CACHE_TIMEOUT), function (req, res, next) {
```
Comment (tgrospic) [inline rchain-coop/src/routes/api.js]: Do we want to be able to check balance only for coop addresses or for any address? I was thinking to create only what is necessary for now.

Comment (zsluedem) [reply rchain-coop/src/routes/api.js]: I can not answer to that question.

**Comment by zsluedem** [comment]
@ian-bloom How to do it next?

**Pull request merged**